packages/design-system/src/styles/components/_Alert.scss

```diff
@@ -3,43 +3,11 @@
 $alert-padding: $spacer-2;
 $alert-bar-size: $spacer-1;
 $alert-icon-size: $spacer-5;
-
-// Links on darker backgrounds that aren't quite dark as inverse
-%link-darker {
-  color: $color-primary-darker;
-  cursor: pointer;
-  text-decoration: underline;
-
-  &:visited {
-    color: $color-visited;
-  }
-
-  &:hover {
-    color: $color-primary-darkest;
-  }
-
-  @if $ds-include-focus-styles {
```

Comment:
Focus style blocks removed because we are no longer conditionally showing and hiding focus styles